_about/release-cadence.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis. Around once a month or so, we take one of these daily
+builds and run it through a number of additional qualification tests and tag the build as an Alpha release.
+Around once a quarter or so, we take one of these Alpha releases, run through a bunch more tests
+and tag the build as a Long Term Support (LTS) release. Finally, if we find something wrong with an Alpha
+or LTS release, we issue patches.
+
+The different types (Daily, Alpha, LTS) represent different product quality levels and different levels of support
+from the Istio team. In this context, *support* means that we will produce patch releases for critical issues and
+offer technical assistance. Separately, 3rd parties and partners may offer longer-term support solutions.
+
+|Type          | Support Level                                         | Quality and Recommended Use
+|--------------+-------------------------------------------------------+----------------------------
+|Daily Build   | No support                                            | Dangerous, may not be fully reliable. Useful to experiment with.
+|Alpha Release | Support is provided for the latest Alpha release only | Expected to be quite stable, but use in production should be limited to an as-needed basis. Usually only adopted by bleeding edge users, or users seeking specific features.
```

Comment:
Are we going to produce release notes for alpha releases going forward?

Reply:
I don't think so, only full notes for LTS releases

Reply:
We should at a minimum point at an automated set of changes between monthlies, but not do a curated set of notes.